shell/platform/android/io/flutter/plugin/platform/PlatformView.java

```diff
@@ -24,4 +25,26 @@
      * after this method is called. Failing to do so will result in a memory leak.
      */
     void dispose();
+
+    /**
+    * Callback fired when the platform's input connection is locked, or should be used. See also
+    * {@link TextInputPlugin#lockPlatformViewInputConnection}.
+    *
+    * <p>This hook only exists for rare cases where the plugin relies on the state of the input
+    * connection. This probably doesn't need to be implemented.
+    */
+    // Default interface methods are supported on all min SDK versions of Android.
```

Comment:
Do we plan to add this comment not to every default method? If not, then we probably shouldn't add it to any.